n3fit/src/n3fit/scripts/n3fit_exec.py

```diff
@@ -158,10 +158,8 @@ def from_yaml(cls, o, *args, **kwargs):
             N3FIT_FIXED_CONFIG['use_scalevar_uncertainties'] = thconfig.get('use_scalevar_uncertainties', True) 
         #Sampling flags
         if (sam_t0:=file_content.get('sampling')) is not None:
-            N3FIT_FIXED_CONFIG['use_t0_sampling'] = sam_t0.get('use_t0', False) 
             N3FIT_FIXED_CONFIG['separate_multiplicative'] = sam_t0.get('separate_multiplicative', True) 
         #Fitting flag
```

Comment:
Is the only other flag of `sampling` `separate_multiplicative` @andreab1997 ? 

If so it might make sense to remove this extra level.

Also, maybe in the same vein as removing the option to not use t0 we might want to remove the option of doing the sampling with the multiplicative uncertainties separated.

Reply:
> Is the only other flag of `sampling` `separate_multiplicative` @andreab1997 ?

To answer this question, yes so it makes sense to remove completely the sampling level

Reply:
I would say let's remove both the sampling level and `separate_multiplicative`